styles.css

```diff
@@ -1,3 +1,7 @@
+body {
+  min-width: 600px;
```

Comment:
Is this just to force the Chrome plugin to a certain width? What happens when you shrink a normal Blinken (ie _not_ the Chrome plugin) to narrower than 600px? Forced x-scroll?

If this line is just for the Chrome plugin, it might be nice to add a plugin-specific CSS file?

Reply:
Yep, just for the plugin. I figured that it doesn't make a difference in other cases, since we're not likely to show it < 600px. Plugin specific CSS would help, but may make this more complicated.

Reply:
Yeah I guess it's fine. I suppose if anyone insists on using this on their low-resolution mobile, we can tackle it then.